### PR TITLE
Add Common PBO

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_drawLine.sqf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* ----------------------------------------------------------------------------
-Function: fza_sfmplus_fnc_drawLine
+Function: fza_fnc_drawLine
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_drawLine.sqf:2]: Maybe a better name would be `drawDebugLine`?

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/CfgFunctions.hpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            class cycle {R;};
+            class drawLine {R;};
+            class getArea {R;};
+            class linearInterp {R;};
+            class padString {R;};
+            class setArrayVariable {R;};
+        };
+        class pid {
+            file = "\fza_ah64_common\functions\pid";
+            class pidCreate {R;};
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_common/CfgFunctions.hpp:22]: So far these functions seem to not defined yet right?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/CfgFunctions.hpp (hunk 2 above)
Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_common/CfgFunctions.hpp:22]: If not, just don't add the files in this PR

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf [hunk 3]
```diff
@@ -38,4 +38,4 @@ switch (_col) do {
     };
 };
 
-drawLine3D [_heli modelToWorldVisual _p1, _heli modelToWorldVisual _p2, _col];
\ No newline at end of file
+debugDrawLine3D [_heli modelToWorldVisual _p1, _heli modelToWorldVisual _p2, _col];
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf:41]: slight error with the replace here

**Review by Ollieollieolllie — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf (hunk 3 above)
Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf:41]: Agreed, I guess this was just a too general find-replace?

Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf]: Fixed!

**Review by Ollieollieolllie — APPROVED** [review]
Looks good

**Review by Ollieollieolllie — APPROVED** [review]
Looks good

**Review by mattysmith22 — APPROVED** [review]

**Pull request merged**